ush/templates/FV3LAM_wflow.xml

```diff
@@ -1013,7 +1016,6 @@ MODULES_RUN_TASK_FP script.
 
   </task>
 
-{%- endif %}
 {%- if machine in ["JET", "HERA"]  %}
```

Comment:
A general question, do retros need the archive task?

Reply:
I am not sure. But I never archived my retros runs and it is OK so far. If we need to archive retro for paper, we should only archive grib2 files under com.